### ISSUE TITLE
Takes a lot of time to answer

### ISSUE THREAD

**Issue by TAPANSHARMA2** [matthewsmkim1/HQ-Bot#2]
Bot is taking on average 8-12 seconds to come up with answer, How can speed it up?

**Comment by matthewsmkim1** [comment]
Yeah, that's something I am trying to figure out right now too. My guess is what is taking the majority of the time are the queries with Google Custom Search API and with Beautiful Soup. I've only been working on this bot for about a week, so hopefully I'll be able to speed up the process.

**Comment by matthewsmkim1** [comment]
Wait, @TAPANSHARMA2, I just saw your other question. Did you figure out how to run it then?

**Comment by TAPANSHARMA2** [comment]
Yes, I got it working now waiting for next update which makes the bot run faster. Thanks

**Comment by shabbir1215** [comment]
does this work on windows as well??
